parseComCode/main.js

```diff
@@ -30,6 +30,141 @@ Parse.Cloud.job("populateSearch", function(request, status) {
   });
 });
 
+//A background job to populate usageCounts for languages and tags
+Parse.Cloud.job("populateCounts", function(request, status) {
+    Parse.Cloud.useMasterKey();
+
+    var counters = { language: {}, tag: {}};
+
+    //Query each tag
+    var tagQuery = new Parse.Query('tag');
+    tagQuery.each(function(tag) {
```

Comment:
According to the Parse documentation: "Because promises know whether they’ve been fulfilled or failed, they can propagate errors, not calling any callback until an error handler is encountered. ... If any Promise in a chain returns an error, all of the success callbacks after it will be skipped until an error callback is encountered. ... It's often convenient to have a long chain of success callbacks with only one error handler at the end."

I can add an error handler to the last then, and it should resolve that problem.